src/commands/interconvert-string-template-literal.ts

```diff
@@ -0,0 +1,51 @@
+import type { Command } from '../types'
+
+const PREFIX = /[\/:@]\s*/
+const STRING_LITERAL = /to-(?:string-literal|sl)|2sl/
+const TEMPLATE_LITERAL = /to-(?:template-literal|tl)|2tl/
+const NUMBER = /\s{0,}(.*)?/
+
+function getNodes<T>(nodes: T[], numbers: number[]) {
+  return numbers.length
+    ? numbers.map(n => nodes[n - 1]).filter(Boolean)
+    : nodes
+}
+
+export const interconvertStringTemplateLiteral: Command = {
+  name: 'interconvert-string-template-literal',
```

Comment:
Ideally I think it would be better to mapping one-command per operation. We could share the implementation, but better to expose multiple command, so users could compose them easier as well.